pkg/cpi/mockingUtils.go

```diff
@@ -26,7 +26,7 @@ func GetCPIFunctionMockResponse(functionName, testType string) (*http.Response,
 			return GetNegativeCaseHTTPResponseBodyAndErrorNil()
 		}
 		return GetParameterKeyMissingResponseBody()
-	case "IntegrationArtifactGetMplStatus":
+	case "IntegrationArtifactGetMplStatus", "APIKeyValueMapDownload":
```

Comment:
CPI is not APIM -> violation of separation of concerns.

Either copy code to pkg/apim and adjust accordingly. 
Or refactor and create a general model.

Reply:
CPI and APIM are parts of same Product Suite, that is SAP Integration Suite.  so common shared library for both does make sense.  it may be either common utility library  or mocking library.